apps/examples/testcase/ta_tc/device_management/utc/utc_dm_main.c

```diff
@@ -40,6 +51,52 @@ extern int working_tc;
 
 static int isConnected = 0;
 
+static slsi_security_config_t *get_security_config(char *sec_type, char *psk)
```

Comment:
@JunhwanPark I did not realize it was renamed the way it is now. Will fix as shown above. Thanks.